anta/tests/logging.py

```diff
@@ -36,22 +42,20 @@ def test(self) -> None:
         """
         Run VerifyLoggingPersistent validation.
         """
-        logger.debug(f"self.instance_commands is: {self.instance_commands}")
-        log_output = cast(str, self.instance_commands[0].output)
-        dir_flash_output = cast(str, self.instance_commands[1].output)
-        logger.debug(f"dataset of log_output command is: {log_output}")
-        logger.debug(f"dataset of dir_flash_output command is: {dir_flash_output}")
-
-        pattern = r"-rw-\s+(\d+)"
-        match = re.search(pattern, dir_flash_output)
-
         self.result.is_success()
 
-        if "disabled" in log_output:
+        output = cast(str, self.instance_commands[0].output)
+
+        if "Persistent logging: disabled" in _get_logging_states(output):
             self.result.is_failure("Persistent logging is disabled")
             return
 
-        if not match or int(match.group(1)) == 0:
+        output = cast(str, self.instance_commands[0].output)
```

Comment:
We have to take the output of the second command here ; `self.instance_commands[1].output`

I think we should keep different output variable names like the initial code so it's not confusing.

Reply:
Right, actually I made a mistake in this code.
Thanks !